mesonbuild/compilers/compilers.py

```diff
@@ -144,6 +144,13 @@ def is_library(fname):
                               'minsize': ['/INCREMENTAL:NO', '/OPT:REF'],
                               }
 
+arm_buildtype_linker_args = {'plain': [],
+                               'debug': [],
+                               'debugoptimized': [],
+                               'release': [],
+                               'minsize': [],
+                               }
```

Comment:
__[Flake8]__

[E124] closing bracket does not match visual indentation


<!-- Comment by [SideCI](https://sideci.com) -->